ui/shared/Input.qml

```diff
@@ -69,7 +69,22 @@ Item {
         height: customHeight
         anchors.top: inputBox.hasLabel ? inputLabel.bottom : parent.top
         anchors.topMargin: inputBox.hasLabel ? inputBox.labelMargin : 0
-
+        Rectangle {
+            id: inputRectangle
+            anchors.fill: parent
+            color: bgColor
+            radius: Style.current.radius
+            border.width: (!!validationError || inputValue.focus) ? 1 : 0
+            border.color: {
+                if (!!validationError) {
+                    return validationErrorColor
+                }
+                if (!inputBox.readOnly && inputValue.focus) {
+                    return Style.current.inputBorderFocus
+                }
+                return Style.current.transparent
+            }
+        }
```

Comment:
This is probably unnecessary when switching to `StatusInput` right away

Reply:
done